Editor/AvatarRigCreator.cs

```diff
@@ -0,0 +1,148 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Animations.Rigging;
+
+namespace ReadyPlayerMe.Core.Editor
+{
+    public abstract class AvatarRigCreator
+    {
+        private const string TAG = nameof(AvatarRigCreator);
+
+        private const string ANIMATION_RIGGING_PACKAGE_NAME = "com.unity.animation.rigging";
+        private const string DIALOG_TITLE = "Read Player Me";
+        private const string DIALOG_MESSAGE = "This will require installing the Animation Rigging package. Do you want to continue?";
+        private const string DIALOG_OK = "Ok";
+        private const string DIALOG_CANCEL = "Cancel";
+
+        private const float HINT_OFFSET_Z = 0.08f;
+
+        [MenuItem("Ready Player Me/Add Avatar Rig")]
```

Comment:
I recommend  changing priority to something like this. When my changes in a separate PR are changed it will go at the bottom section with networking stsuff like this which I think is nicer. Open to discuss of course :D 

![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/9f8c6944-9a74-4e1f-9140-00ab52b16ea5)

```suggestion
        [MenuItem("Ready Player Me/Add Avatar Rig", priority = 35)]
```